### PR TITLE
Adds '/2/files/copy_batch/check_v2' endpoint

### PR DESCRIPTION
**Pull request by anshu1992**
**ENDPOINT:** https://www.dropbox.com/developers/documentation/http/documentation#files-copy_batch-check

### CODE REVIEW

File: lib/dropbox_api/results/copy_batch_check_result.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module DropboxApi::Results
+  class CopyBatchCheckResult < DropboxApi::Results::Base
```
Comment (Jesus) [inline lib/dropbox_api/results/copy_batch_check_result.rb]: I think it makes more sense to name this `RelocationBatchV2Result` or `RelocationBatchResult` so it'd map directly with Dropbox API.

Another benefit is that it won't be too confusing when we re-use this for `move_batch`.

File: lib/dropbox_api/endpoints/files/copy_batch_check.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    ErrorType   = DropboxApi::Errors::PollError
+
+    # 
+    # @param async_job_id [String] Id of the asynchronous job. 
+    #   This is the value of a response returned from the method that launched the job.
+    # 
+    # @return Returns the status of an asynchronous job for copy_batch:2. 
+    # It returns list of results for each entry.  
+    #
+
```
Comment (Jesus) [inline lib/dropbox_api/endpoints/files/copy_batch_check.rb]: Could you update the documentation of the endpoint?

Normally I just copy the wording from Dropbox API docs, then adjust the bits that need to so it makes sense for the person who'll be using this library. Finally trim the lines that exceed 80 cols.

File: lib/dropbox_api/results/copy_batch_check_result.rb (hunk 1 above)
Comment (anshu1992) [reply lib/dropbox_api/results/copy_batch_check_result.rb]: Ah okay, making that change, thanks :)

File: lib/dropbox_api/endpoints/files/copy_batch_check.rb (hunk 2 above)
Comment (anshu1992) [reply lib/dropbox_api/endpoints/files/copy_batch_check.rb]: Updating it :)

**Comment by anshu1992** [comment]
Hi Jesus, I have made the changes you requested. I was not sure about updating the api documentation comments. So I have made the changes by copying some more details from the dropbox documentation. If I have missed something or you need more changes, please let me know. Thanks :)

**Pull request merged**